scripts/etc/udhcpc/default.script

```diff
@@ -19,7 +19,7 @@ PEERDNS_IF=eth0
 case "$1" in
   deconfig)
     # bring interface up, but with no IP configured:
-    ip addr flush dev $interface
+    ip -f inet addr flush dev $interface
     ip link set $interface up
```

Comment:
Thanks, makes a lot of sense :+1: 
The man page wasn't exactly clear on this ... and as I still don't have IPv6 myself, I couldn't test. (I should fix that though). So there was/is the possibility that it would be possible for a dual-stack-ed device.